docs/guides/getting-started-java-spring.md

```diff
@@ -82,22 +82,23 @@ Next, we'll create a BPMN diagram to represent the transaction model shown at th
    2. **Script/Result variable**: `totalWithTax`
    3. **Script/FEEL expression**: `total * 1.1` (this represents the tax applied to the transaction.)
 7. Append a task named `Charge credit card`.
-8. Click on the task and click the wrench-shaped icon to change the type of task to a service task. In the properties panel, change the **Task definition/Type** to `charge-credit-card`.
+8. Click on the task and click the wrench-shaped icon to change the type of task to a service task. In the properties panel, change the **Task definition/Job type** to `charge-credit-card`.
 9. Append an end event named `Payment executed`.
 10. Save this BPMN file to your Spring project in `src/main/resources`, and name it `process-payments.bpmn`.
 
-## Step 3: Deploy your process
+## Step 3: Deploy your process from Desktop Modeler
 
 To deploy your process, take the following steps:
 
-1. Open Desktop Modeler and click the rocket icon in the bottom left corner.
-2. Change the **Deployment name** to `process-payments`, and ensure the **Target** is `Camunda 8 Self-Managed`.
-3. Change the **Cluster endpoint** to `http://localhost:26500/`, with no authentication.
-4. Click **Deploy**.
+1. Open Desktop Modeler and ensure the desired Camunda8 version is targeted in the bottom left corner.
```

Comment:
Especially with two different alphas available, this feels like an easy thing to mess up.